fuel-relayer/README.md

```diff
@@ -25,105 +18,39 @@ Example of sliding window:
 ![Sliding Window](../docs/diagrams/fuel_v2_relayer_sliding_window.jpg)
 
 * Problem: How to choose when bridge message event gets enabled for use in fuel, at what exact fuel block does this happen? (Note that we have sliding window)
-* Solution: introduce `da_height` variable inside fuel block header that will tell at what block we are including validator delegation/withdrawal and token deposits.
+* Solution: introduce `da_height` variable inside fuel block header that will tell at what block we are including token deposits.
 
-There are few rules that `da_height` (da as data availability) need to follow and can be enforced with v2 contract:
+There are few rules that `da_height` need to follow and can be enforced with v2 contract:
 
 * Last block `da_height` needs to be smaller number then current one.
 * `da_height` shouldn't be greater then `best_da_block`-`finalization_slider`
 
 In most cases for fuel block `da_height` can be calculated as `current_da_height-1`
```

Comment:
@Voxelot Is this still relevant / correct? I don't really understand what it means.

Reply:
I think this would be more of a concern for the block producer when selecting a da_height. Since we explicitly include a da_height on our block headers, we shouldn't need to make any guesses about the da_height in our code. I vote for just removing this.